### PR TITLE
first draft RN

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- RN for 2017.35

### Resolves:

* [OKTA-138664](https://oktainc.atlassian.net/browse/OKTA-138664)

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
@jmelberg-okta I am getting a Gemfile.lock Travis failure even after adding the Gemfile.lock per the instructions in the travis log. It says I deleted "json" from the file, which seems weird (and I didn't do on purpose). Any suggestion for what to try next? Thanks! I had hoped to push this out before I get on a plane at 11:30 AM Thursday.

File: Gemfile.lock [hunk 1]
```diff
@@ -187,6 +188,7 @@ GEM
       gemoji (~> 3.0)
       html-pipeline (~> 2.2)
       jekyll (>= 3.0)
+    json (2.1.0)
```
Comment (jmelberg-okta) [inline Gemfile.lock]: I believe `json` is no longer a required dependency. Is this the file that is commited after rebasing off of our current `master` branch?

Comment (mystiberry-okta) [reply Gemfile.lock]: I will try that and add to my troubleshooting list. thx!

**Comment by mystiberry-okta** [comment]
@jmelberg-okta This is not right. My flight is boarding soon, I'll ping you when I get home.

**Comment by mystiberry-okta** [comment]
@jmelberg-okta when it passes travis, ready for review!

**Comment by mystiberry-okta** [comment]
gemfile lock no longer has JSON, @jmelberg-okta

**Pull request merged**